src/WebJobs.Script.WebHost/Management/LinuxSpecialization/PackageDownloadHandler.cs

```diff
@@ -214,5 +229,57 @@ private async Task<bool> IsResourceAccessibleWithoutAuthorization(string resourc
                 return false;
             }
         }
+
+        private string CopyPackageFile(IFileSystem fileSystem = null)
+        {
+            fileSystem ??= FileUtility.Instance;
```

Comment:
lets inject via constructor and make _fileSystem member variable and do this fall back logic in constructor